cypress/support/awx-commands.ts

```diff
@@ -514,6 +514,15 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add(
+  'clickTableRowRoleAction',
```

Comment:
just curious why not use `clickTableRowPinnedAction` or `clickTableRowAction`, they look very similar to this new one

Reply:
@acruzgon good catch, we have so many custom commands now 😄 always good to know a new one
updating the command.